Frontend/ui-customer/src/Flow.purs

```diff
@@ -1470,6 +1472,22 @@ homeScreenFlow = do
         pure unit
       homeScreenFlow
     REFRESH_HOME_SCREEN -> homeScreenFlow
+    TRACKING_RESTART -> do 
```

Comment:
can we call REFRESH_HOME_SCREEN instead of tracking_restart and set the storage key in controller.